pkg/solana/logpoller/orm.go

```diff
@@ -156,3 +157,27 @@ func (o *DSORM) SelectLogs(ctx context.Context, start, end int64, address Public
 	}
 	return logs, nil
 }
+
+func (o *DSORM) FilteredLogs(ctx context.Context, filter []query.Expression, limitAndSort query.LimitAndSort, _ string) ([]Log, error) {
```

Comment:
I was thinking the `FilteredLogs` method that calls this would also be in this PR, but I guess it makes more sense if I just include it in my Main LogPoller Loop PR... probably too difficult to add it here without the struct that's defined in RegisterFilters PR.

Reply:
Oh, I do think we should add some tests of this function to `orm_tests.go`.

In evm, there are several tests that test it in different ways:  `TestORM_IndexedLogs`,  `TestORM_SelectLogsCreatedAfter`, `TestORM_SelectIndexedLogsByTxHash`, `TestORM_SelectLogsWithSigsByBlockRangeFilter`. Those also test the corresponding legacy functions which we don't need. But maybe we could just combine the parts that test `FilterLogs` together into a single `TestFilteredLogs` here?